packages/types/src/Quote.ts

```diff
@@ -28,5 +28,6 @@ export interface IQuoteAttesterSigned extends IQuote {
 
 export interface IQuoteAgreement extends IQuoteAttesterSigned {
   rootHash: ICredential['rootHash']
+  claimerDid: DidUri
```

Comment:
Doesn't `DidSignature` already contains the key URI? Do we need also the `claimerDid`?

Reply:
The point is that because the key URI is part of the signature, it is not signed into the credential... so you could change the claiming party after the fact, as long as the key exists on the other DID document. And the claimerDid is the one that is supposed to go into the credential, so forcing claimers to commit to an identity in the agreement seems to make sense